go/ephemeral/lib.go

```diff
@@ -65,7 +65,7 @@ func NewEKLib(mctx libkb.MetaContext) *EKLib {
 		stopCh:                 stopCh,
 	}
 	if !mctx.G().GetEnv().GetDisableEKBackgroundKeygen() {
-		go ekLib.backgroundKeygen(mctx, stopCh)
+		go ekLib.backgroundKeygen(mctx.BackgroundWithLogTags(), stopCh)
```

Comment:
some preexisting bugs where we didn't correctly use a background context in this file